app/layout.tsx

```diff
@@ -0,0 +1,14 @@
+import "../styles/globals.css";
+
+type Props = {
+  children: React.ReactNode;
+};
+const RootLayout = ({ children }: Props) => {
+  return (
+    <html>
```

Comment:
이렇게 html의 lang 속성도 적용해주면 좋을 것 같아요~
```suggestion
    <html lang="ko">
```